integration/grafana/README.md

```diff
@@ -1,25 +1,147 @@
-
-
-Importing the Grafana dashboard
--------------------------------
-Follow the steps documented in the official Grafana user guide
-<https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/import-dashboards/>
-to import any of the JSON files in this directory for generating
-the preferred visualizations. 
-
-
-Data Source for the Visuals
----------------------------
-Grafana is capable of sourcing telemetry data from Prometheus
-clients running on remote hosts. One such Prometheus exporter
-is the `geopmexporter(1)` 
+# Grafana
+
+This directory contains a JSON file for the GEOPM Grafana dashboard that
+visualizes metrics generated by the `geopmexporter(1)` Prometheus data source.
+Additionally the directory contains a script to enable the monitoring of jobs
+launched using the PBS HPC resource manager.
+
+
+## Importing the Grafana dashboard
+
+Follow the steps documented in the official Grafana user guide on [importing
+dashboards](https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/import-dashboards)
+to import the GEOPM Report JSON file in this directory.  This will enable the
+visualization of data collected by the `geopmexporter(1)`.
+
+
+## Data Source for the Visuals
+
+Grafana is capable of sourcing telemetry data from Prometheus clients running on
+remote hosts. One such Prometheus exporter is the `geopmexporter(1)`
 <https://geopm.github.io/geopmexporter.1.html> which leverages
-`geopm.stats.collector` to summarize the metrics. The dashboard
-file - `GEOPM_Report.grafana.dashboard.json`, in this directory, 
-relies on `geopmexporter(1)` for its data source.
+`geopm.stats.collector` to summarize the metrics. The dashboard file -
+`GEOPM_Report.grafana.dashboard.json`, in this directory, relies on
+`geopmexporter(1)` for its data source.
+
+
+## Use case: Monitoring PBS jobs
+
+The `geopmexporter(1)` is typically deployed system-wide with either a [systemd
+service](https://geopm.github.io/geopmexporter.1.html#systemd-service) or as a
+[kubernetes service](https://github.com/cmcantalupo/geopm/tree/prometheus/integration/k8#deploying-prometheus-client-in-kubernetes).
+This documentation is for an alternative `geopmexporter(1)` deployment scenario
+where an unprivileged user of a PBS managed system would like to monitor the
+compute resources granted to the user only while they are allocated.  This
+methodology should not be applied for system-wide monitoring by an
+administrator, but rather for job monitoring by an end user of a PBS system
+where the geopmexporter is not deployed system-wide.
+
+A script is provided called `pbs_prometheus_launch.py` that can be used to:
+
+- Assist with the initial one-time configuration of the Grafana server
+- Monitor jobs submitted to a PBS batch system
+- Review data collected from previously monitored jobs
+
+The script runs the Prometheus and Grafana server on the head node of a
+system. Optionally, the script may also deploy and connect to `geopmexporter(1)`
```

Comment:
The way this is worded makes it sound like monitoring the nodes allocated to a user's job is a secondary purpose of the script, when actually, this is its main purpose. Given all the goals/functions of the script are already enumerated above, it seems the goal of this paragraph is just to mention the "how". A suggestion:

"To perform these functions, the script runs the Prometheus and Grafana servers on the head node of a system."